pkg/registry/csv.go

```diff
@@ -341,3 +344,26 @@ func (csv *ClusterServiceVersion) GetOperatorImages() (map[string]struct{}, erro
 
 	return images, nil
 }
+
+// GetSubstitutesFor returns the name of the ClusterServiceVersion object that
+// is substituted by this ClusterServiceVersion object.
+//
+// If not defined, the function returns an empty string.
+func (csv *ClusterServiceVersion) GetSubstitutesFor() (string, error) {
```

Comment:
This won't find substitutes-for if it's in a property, right?

Reply:
No this change is purely on the CSV. We'll have to make a change when csvless bundles arrive to make substitutes for a field in the properties.yaml file.

Reply:
Do we envision substitutes-for being a permanent graph-altering property? (I hope not)

If we can express substitutes-for semantics with existing graph-defining fields (e.g. replaces/skips), then I don't think we want yet another field thrown into the mix.

See my related comment: https://github.com/operator-framework/operator-registry/pull/612#discussion_r605923786

Reply:
what do you mean by permanent graph altering property? I expect the substitution to take affect until a new substitution or bundle defines a different graph just like replaces / skips works today.